src/@chakra-ui/gatsby-plugin/components/Tag/index.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  createMultiStyleConfigHelpers,
+  theme,
+  defineStyle,
+} from "@chakra-ui/react"
+import { tagAnatomy } from "@chakra-ui/anatomy"
+import { $badgeColor, STATUS_COLORS } from "./utils"
+import { defineMergeStyles } from "../components.utils"
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(tagAnatomy.keys)
+
+const { Tag: tagTheme } = theme.components
+
+const baseStyleContainer = defineMergeStyles(tagTheme.baseStyle?.container, {
+  border: "1px",
+  borderColor: "transparent",
+  gap: 1,
+  borderRadius: "full",
+  px: 2,
+  minH: 8,
+  fontWeight: 300,
+  "&:any-link": {
+    textDecor: "none",
+    _focusWithin: {
+      outline: "4px solid",
+      outlineColor: "transparent",
+      outlineOffset: 0,
+    },
+  },
+})
+
+const baseStyleLabel = defineStyle({
+  ...tagTheme.baseStyle?.label,
+  fontSize: "xs",
+  textTransform: "uppercase",
+  textAlign: "center",
+  lineHeight: 1,
+})
+
+const baseStyleCloseButton = defineStyle({
+  ...tagTheme.baseStyle?.closeButton,
+  opacity: 1,
+  m: 0,
+  // Clear default
+  _focusVisible: null,
+  "&:focus-visible, &:hover": {
+    outline: "3px solid",
+    outlineOffset: "-2px",
+  },
```

Comment:
Cheap solution to get focus/hover styling.
1. override default styling by nulling out (because I'm using a multi-selector instead)
2. Provide a more generous styling of an outline ring (this is taking advantage of the default `outlineColor` value of `currentColor`)

Reply:
Nice! looks great now